components/dashboard/src/settings/Teams.tsx

```diff
@@ -527,6 +528,16 @@ function AllTeams() {
                 <div className="flex-grow ">
                     <h3 className="self-center">All Teams</h3>
                     <h2>Manage teams and team members.</h2>
+                    <ItemsList className="pt-2">
+                        <Item className="w-full bg-yellow-200 hover:bg-yellow-200 py-6 px-6">
+                            <ItemField className=" flex flex-col">
+                                <div className="text-yellow-600 font-semibold">DEPRECATION NOTICE</div>
+                                <p className="text-yellow-800">
+                                    Hey! We're introducing <strong>Teams & Projects</strong> to surface <strong>Prebuilds</strong> in the dashboard. All existing teams will be migrated over the upcoming weeks. There's no action required on your side. <a href="/new" className="learn-more">Learn more about Teams & Projects</a> or <a href="/new" className="learn-more">Create a New Team</a>.
```

Comment:
**issue(blocking)**: The first link here should link to some documentation which does not exist yet. Alternatively, We could drop the last sentence here for now and add it back once we have some documentation we can link to.

 ```suggestion
                                    Hey! We're introducing <strong>Teams & Projects</strong> to surface <strong>Prebuilds</strong> in the dashboard. All existing teams will be migrated over the upcoming weeks. There's no action required on your side.
```